README.md

```diff
@@ -1,2 +1,85 @@
+|../DPC++FPGA/Tutorials/Features/loop_unroll                          |FPGA |Tutorial demonstrating the DPC++ unroll pragma and its performance trade-offs on FPGA |
+|../DPC++FPGA/Tutorials/Features/max_concurrency                      |FPGA |Tutorial demonstrating the DPC++ FPGA max_concurrency attribute |
+|../DPC++FPGA/Tutorials/Features/memory_attributes                    |FPGA |Tutorial demonstrating how to use DPC++ FPGA memory attributes |
+|../DPC++FPGA/Tutorials/Features/pipes                                |FPGA |Tutorial demonstrating the DPC++ FPGA pipes extension to transfer data between kernels |
+|../DPC++FPGA/Tutorials/Features/speculated_iterations                |FPGA |Tutorial demonstrating the DPC++ FPGA speculated_iterations attribute |
+|../C++/CombinationalLogic/Mandelbrot   	|CPU  	|Demonstrates how to accelerate   Mandelbrot performance with SIMD and parallelization using OpenMP*.   	|
+|../C++/CompilerInfrastructure/Intrinsics    	|CPU  	|Shows how to utilize the   intrinsics supported by C++ compiler in a variety of applications.	|
+|../C++/GraphTraversal/Mergesort   	|CPU  	|Shows how to accelerate scalar   merge sort program using OpenMP tasks   	|
+|Libraries |
+|../oneDPL/Gamma-correction 	|GPU, CPU  	|gamma correction using Parallel   STL 	|
+|../oneDPL/Stable_sort_by_key    	|GPU, CPU  	|stable sort by key using   counting_iterator and zip_iterator  	|
+|../oneVPL/hello-decode	|CPU  	|shows how to use oneVPL to   perform a simple video decode	|
+|../oneVPL/hello-encode	|CPU  	|shows how to use oneVPL to   perform a simple video encode	|
+|Tools |
+|../ApplicationDebugger/Debugger/array-transform    	|GPU, CPU  	|Array transform   	|
+|../IoTConnectionTools/Analog-in	|CPU  	|Analog   pin input example using Eclipse* MRAA  	|
+|../IoTConnectionTools/Digital   In  	|CPU  	|GPIO   pin input example using Eclipse* MRAA    	|
+|../IoTConnectionTools/Digital   Out 	|CPU  	|GPIO   pin output example using Eclipse* MRAA   	|
+|../IoTConnectionTools/Hello   IoT World  	|CPU  	|Basic   example that prints the compiler used during build	|
+|../IoTConnectionTools/Interrupt	|CPU  	|Interrupt   Service Routine example using Eclipse* MRAA   	|
+|../IoTConnectionTools/Onboard   Blink    	|CPU  	|Built-in   LED blink for common IoT boards using Eclipse* MRAA 	|
+|../IoTConnectionTools/PWM 	|CPU  	|Pulse   Width Modulation pin output using Eclipse* MRAA   	|
+|../IoTConnectionTools/Up2   LEDs    	|CPU  	|Built-in   LED example for UP* Squared using Eclipse* MRAA	|
+|../SystemDebug/System Debug   Sample Build    	|UEFI 	|Basic example that   showcases the features of the Intel® System Debugger	|
+
+#License
+ 
+The code samples are licensed under MIT license
+ 
+#Known issues or limitations
+ 
+##On Windows Platform
+1. If you are using Visual Studio 2019, Visual Studio 2019 version 16.3.0 or newer is required.
+2. To build samples on Windows, the required Windows SDK is ver. 10.0.17763.0. 
```

Comment:
This whole bullet can be removed as we no longer depend on specific SDK versions

Reply:
@JoeOster  you marked this as resolved but the change has not been made. Can you confirm that this is indeed resolved.